databend-client/src/main/java/com/databend/client/data/Int8Handler.java

```diff
@@ -604,3 +605,62 @@ public void setNullable(boolean isNullable) {
         this.isNullable = isNullable;
     }
 }
+
+
+class DecimalHandler implements ColumnTypeHandler {
```

Comment:
@raymondchen-byte Thanks for your contributions! Would you like to add some tests about `DecimalHandler`?